app/src/main/res/values/bools.xml

```diff
@@ -2,9 +2,11 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <bool name="theme_is_light">true</bool>
     <bool name="API28">false</bool>
     <bool name="underAPI28">true</bool>
     <bool name="tablet">false</bool>
+    <!-- disable the automatic install of Leak Canary -->
+    <bool name="leak_canary_watcher_auto_install" tools:ignore="UnusedResources">false</bool>
```

Comment:
You can remove this and depend on leakcanary-android-core instead of leakcanary-android

https://square.github.io/leakcanary/changelog/#androidx-app-startup